lib/std/posix/test.zig

```diff
@@ -1180,6 +1180,27 @@ test "POSIX file locking with fcntl" {
     }
 }
 
+test "posix getpid" {
+    if (!@hasDecl(posix.system, "getpid"))
+        return error.SkipZigTest;
```

Comment:
Should be replaced with appropriate target checks.

Reply:
Not all too sure what the behaviour is when _not_ linking libc for a majority of supported targets, though. Time to wait and see the results of the tests?